pkg/virt-handler/vm_test.go

```diff
@@ -1131,13 +1130,13 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 				vmiFeeder.Add(vmi)
 
-				fakeDiskChecksums := &container_disk.DiskChecksums{
+				fakeDiskChecksums := &containerdisk.DiskChecksums{
 					ContainerDiskChecksums: map[string]uint32{
 						vmi.Spec.Volumes[0].Name: uint32(1234),
 					},
-					KernelBootChecksum: container_disk.KernelBootChecksum{
-						Kernel: pointer.Uint32(33),
-						Initrd: pointer.Uint32(35),
+					KernelBootChecksum: containerdisk.KernelBootChecksum{
+						Kernel: virtpointer.P(uint32(33)),
```

Comment:
We usually just call it `pointer`, I don't think the name `virtpointer` is necessary now that the k8s pointers are gone.